rootfs/etc/os-release

```diff
@@ -0,0 +1,9 @@
+NAME="Chimera OS"
+PRETTY_NAME="Chimera"
```

Comment:
Should include the release (you can get it as we tag each pre-release with the number and commit). Maybe you can do a @@ substitution in the build process.